gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/security/jwt/JWTAuthenticator.java

```diff
@@ -202,11 +200,9 @@ public AuthenticationContext authenticate(RequestContext requestContext) throws
 
                     // Validate subscriptions
                     APIKeyValidationInfoDTO apiKeyValidationInfoDTO = new APIKeyValidationInfoDTO();
-                    EnforcerConfig configuration = ConfigHolder.getInstance().getConfig();
-                    ExtendedTokenIssuerDto issuerDto = configuration.getIssuersMap().get(validationInfo.getIssuer());
                     Scope validateSubscriptionSpanScope = null;
                     try {
-                        if (issuerDto.isValidateSubscriptions() && !requestContext.getMatchedAPI().isSystemAPI() ) {
+                        if (false) {
```

Comment:
shall we add todo comments in such places where we have added temporary changes, so we can identify them easily? 
// TODO(TharinduD) revisit when subscription validation is enabled
```suggestion
                        if (false) {
```